File: src/web_app/case.js
class Case {
    constructor(caseNumber, caseValue) {
        this.caseNumber = caseNumber;
        this.caseValue = caseValue;
        this.isOpen = false;
    }

    openCase() {
        this.isOpen = true;
    }

    getCaseValue() {
        return this.caseValue;
    }

    isCaseOpen() {
        return this.isOpen;
    }
}

export default Case;

File: src/case.py
class Case:
    def __init__(self, case_number, case_value):
        self.case_number = case_number
        self.case_value = case_value
        self.is_open = False

    def open_case(self):
        self.is_open = True

    def get_case_value(self):
        return self.case_value

    def get_case_number(self):
        return self.case_number

    def is_case_open(self):
        return self.is_open